products/compute/ansible.yaml

```
@@ -216,6 +216,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         aliases: ["ip_forward"]
       disks.initializeParams.sourceImage: !ruby/object:Overrides::Ansible::PropertyOverride
         aliases: ["image", "image_family"]
+      networkInterfaces.network: !ruby/object:Overrides::Ansible::PropertyOverride
+        description: |
+          Specifies the title of an existing network. Not setting the network
+          title, will select the default network interface, which could have
```

nit: remove the comma between "title" and "will"
```suggestion
          title will select the default network interface, which could have
```